uploadNewMarcxmlRecords.py

```diff
@@ -0,0 +1,56 @@
+from internetarchive import get_session
+import argparse
+import csv
+import os
+import shutil
+import datetime
+import secrets
+
+# command line arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('-f', '--fileName', help='the file of data to be searched. \
+optional - if not provided, the script will ask for input')
+args = parser.parse_args()
+
+if args.fileName:
+    fileName = args.fileName
+else:
+    fileName = input('Enter the file of data to be searched: ')
+
+# archive.org authentication
+c = {'s3': {'access': secrets.access, 'secret': secrets.secret}}
+s = get_session(config=c)
+
+# get row count
+with open(fileName) as csvfile:
+    reader = csv.DictReader(csvfile)
+    rowCount = len(list(reader))
+
+# create log
+f = csv.writer(open('recordsUpdated.csv', 'w'))
+f.writerow(['iaID'] + ['oclcNum'] + ['response'] + [''])
+
+# script content
+backupDirectory = 'archive.orgBackup/'
+replacementDirectory = 'oclcRecords/'
+if os.path.isdir('archive.orgBackup/') is False:
+    os.makedirs('replacementDirectory')
```

Comment:
Should this be a string? My assumption is you want the `replacementDirectory` variable here.

Reply:
Good catch! That was a holdover before I made those folder names variables. I just updated it